Obsidian.ConsoleApp/Program.cs

```diff
@@ -1,177 +1,41 @@
-﻿using Obsidian.Utilities;
-using System.Globalization;
-using System.Reflection;
-using System.Runtime.InteropServices;
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Obsidian.ConsoleApp.Logging;
+using Obsidian.Hosting;
+using Obsidian.Utilities;
 
 namespace Obsidian.ConsoleApp;
 
-public static class Program
+public static partial class Program
```

Comment:
Any specific reason why you changed this to be partial? Where are the other parts?

Reply:
This is a mistake on my part, I don't even remember doing that...